scripts/falco-driver-loader

```diff
@@ -300,13 +300,13 @@ clean_kernel_module() {
 		return
 	fi
 
-	DRIVER_VERSIONS=$(dkms status -m "${DRIVER_NAME}" | cut -d',' -f2 | sed -e 's/^[[:space:]]*//')
+	DRIVER_VERSIONS=$(dkms status -m "${DRIVER_NAME}" | cut -d',' -f1 | sed -e 's/^[[:space:]]*//')
 	if [ -z "${DRIVER_VERSIONS}" ]; then
 		echo "* There is no ${DRIVER_NAME} module in dkms"
 		return
 	fi
 	for CURRENT_VER in ${DRIVER_VERSIONS}; do
-		if dkms remove -m "${DRIVER_NAME}" -v "${CURRENT_VER}" --all 2>/dev/null; then
+		if dkms remove "${CURRENT_VER}" --all 2>/dev/null; then
 			echo "* Removing ${DRIVER_NAME}/${CURRENT_VER} succeeded"
```

Comment:
```suggestion
			echo "* Removing ${CURRENT_VER} succeeded"
```

Since `CURRENT_VER` should already contain something like `falco/a743aa4ab28e4bd82c73f74be4f866911a4be0fa` so we don't need to prefix it.